lib/Conversion/XTenToLinalg.cpp

```diff
@@ -70,20 +69,22 @@ static Value applyPad(Location loc, Value input, ArrayRef<int64_t> pad,
       .result();
 }
 
+/// Return a zero-initialized tensor of given size and dtype.
+static Value zeroInit(ArrayRef<int64_t> sizes, mlir::Type elementType, Location loc,
+                         ConversionPatternRewriter &rewriter) {
+  Value initTensor = rewriter.create<linalg::InitTensorOp>(loc, sizes, elementType);
+  Value c0float = rewriter.create<arith::ConstantOp>(
+      loc, FloatAttr::get(elementType, 0.0));
```

Comment:
```suggestion
      loc, rewriter.getZeroAttr(elementType));
```

Reply:
Also, `getZeroAttr` even works for full tensors (assuming only static sizes). I'm not sure this is applicable here, however, because `sizes` may contain `ShapedType::kDynamicSize` I guess.